app/labor/emoji_converter.rb

```diff
@@ -11,7 +11,7 @@ def initialize(html)
 
   def convert
     html.gsub!(/:([\w+-]+):/) do |match|
-      emoji = Emoji.find_by_alias($1)
+      emoji = Emoji.find_by_alias(Regexp.last_match(1))
```

Comment:
Mmm, I think this should be `Regexp.last_match(0)`

`$1` is the first matching group, see [special global variables](https://ruby-doc.org/core-2.6.1/Regexp.html#class-Regexp-label-Special+global+variables):

> $1, $2 and so on contain text matching first, second, etc capture group;

`Regex.last_match(1)` instead returns the match at the index `1` which is the second match, not the first one. The correct change should be `Regex.last_match(0)`

Reply:
Okay. Ruby Style guide shows it to be `Regexp.last_match(1)`

Reply:
Same as in `markdown_fixer`. Should be changed back to `Regexp.last_match(1)`